lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<blockquote>The PyMedPhys&nbsp;<a href="https://github.com/pymedphys/pymedphys/raw/c029675cd7d66b663c9ab1971b55fa38be5c38cc/lib/pymedphys/docs/lib/ref/gamma.rst" target="_blank">gamma</a>&nbsp;<strong>tool </strong>allows <strong>the user</strong> to compute <s>the </s>gamma (γ) index <strong>distributions </strong>between a reference and an evaluation dose distribution in 1D, 2D and 3D <strong>and obtain gamma pass rates for these distributions</strong>. <strong>The tool features several options for optimising the computation for the user's application.</strong></blockquote><p> </p><blockquote>In this tutorial, you will compute the <strong>gamma </strong>pass rate using the PyMedPhys <a href="https://github.com/pymedphys/pymedphys/raw/c029675cd7d66b663c9ab1971b55fa38be5c38cc/lib/pymedphys/docs/lib/ref/gamma.rst" target="_blank">gamma</a> <strong>tool</strong>. You will tweak its parameters to speed up the computation of the gamma index <strong>distribution</strong>.</blockquote><p> </p><p>Also, your links to "PyMedPhys gamma" throughout the tutorial probably need to be directed to the published <a href="https://docs.pymedphys.com/lib/ref/gamma.html" target="_blank">docs</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>